src/components/SearchSploit/SearchSploit.tsx

```diff
@@ -142,122 +122,124 @@ const SearchSploit = () => {
-                                <List.Item>Overflow | Exploit titles are allowed to overflow their columns.</List.Item>
-                                <List.Item>
-                                    Path | Show the full path to an exploit (and also copies the path to the clipboard
-                                    if possible).
-                                </List.Item>
-                                <List.Item>Verbose | Display more information in output.</List.Item>
-                                <List.Item>www | Show URLs to Exploit-DB.com rather than the local path.</List.Item>
-                            </List>
-                            <br></br>
-                            <List>
-                                <Text weight={700}>Non-Searching:</Text>
-                                <List.Item>
-                                    Mirror | Mirror (aka copies) an exploit to the current working directory.
-                                </List.Item>
-                                <List.Item>Examine | Examine (aka opens) the exploit using $PAGER.</List.Item>
-                            </List>
-                        </Accordion.Panel>
-                    </Accordion.Item>
-                </Accordion>
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-            </Stack>
-        </form>
+            )}
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                <Stack>
+                    {LoadingOverlayAndCancelButtonPkexec(loading, pid, handleCancel, handleProcessTermination)}
+                    <TextInput label={"Search Term"} {...form.getInputProps("searchTerm")} />
+                    <NativeSelect
+                        {...form.getInputProps("searchOption")}
+                        label={"Search Option"}
+                        data={searchOptions}
+                        placeholder={"Pick a Search option"}
+                    />
+                    <NativeSelect
+                        {...form.getInputProps("outputType")}
+                        label={"Output"}
+                        data={outputTypes}
+                        placeholder={"Select an Output"}
+                    />
+                    <NativeSelect
+                        {...form.getInputProps("nonSearch")}
+                        label={"Non-Searching"}
+                        data={nonSearchOptions}
+                        placeholder={"Select an option"}
+                    />
+                    <TextInput
+                        label={"EBD-ID"}
+                        description="Exploit Database ID: Required when using the 'Path' output or Non-Search options."
+                        {...form.getInputProps("ebdId")}
+                    />
+                    <Button type={"submit"}>Scan</Button>
```

Comment:
You should use Start {title}
```suggestion
                    <Button type={"submit"}>Start {title}</Button>
```